app/controllers/rent_controller.rb

```diff
@@ -0,0 +1,15 @@
+class RentController < ApplicationController
+  include Wor::Paginate
+  include DeviseTokenAuth::Concerns::SetUserByToken
+
+  def create
+    Rent.create(user_id: params[:user], book_id: params[:book],
+                initial_date: params[:idate], final_date: params[:fdate])
```

Comment:
To ease the understanding of the code and allow only parameters we what when we are doing and indestion in the DB or and update, we use permit method form ActionContoller.
I give you here an example:
```ruby
def create
  Rent.new(create_params)
  # folow the logic of you controller.
end

private

def create_params
  params.permit(:your, :permited, :params)
end